submissioncclicenses.md

```diff
@@ -115,11 +115,27 @@ Some licenses contain questions, e.g. standard retrieved from https://api.creati
 The fields are questions to be answered when assigning the license
 
 ## Search CC License 
-**/api/config/submissioncclicenses/search/rightsByQuestions**
+**/api/config/submissioncclicenseurl/search/rightsByQuestions**
+
+The endpoints above don't return the url that will also be needed to add the license to the metadata. 
+It cannot be provided in the endpoints above as the url can differ depending on the answers that are provided. The answers are only required if the endpoints above return them.
 
 Parameters:
 * license: the ID of the license (e.g. standard, publicdomain, …)
 * answer_X: List of answers (e.g. answer_commercial=y&answer_derivatives=sa)
```

Comment:
How does a client determine how to name these params? Do these need to correspond to `field` ID values from the `/api/config/submissioncclicenses/<:license-id>` endpoint?  If so, we need to document that here by saying something like "X must be a valid field ID returned from `/api/config/submissioncclicenses/<:license-id>`, and the answer value must be a valid value ID for that field."

Reply:
You are correct, I made the change, I also added the error codes that can be returned by this endpoint, this should make things more clear what happens if you provide invalid input.